controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -155,6 +155,35 @@ func (r *StorageConsumerReconciler) reconcileEnabledPhases() (reconcile.Result,
 		if err := r.deleteOnboardingSecret(); err != nil {
 			return reconcile.Result{}, nil
 		}
+
+		storageCluster, err := util.GetStorageClusterInNamespace(r.ctx, r.Client, r.namespace)
+		if err != nil {
+			return reconcile.Result{}, err
+		}
+
+		// isCephFsAvailable - get the default filesystem, isRbdAvailable - list blockpools CR with count 1, isNfsAvailable -
```

Comment:
If you want to mentioned the criteria in a comment, please do it per service above the section of code that deal with that service